src/pykeen/pipeline/api.py

```diff
@@ -1589,70 +1585,3 @@ def pipeline(  # noqa: C901
         train_seconds=train_seconds,
         evaluate_seconds=evaluate_seconds,
     )
-
-
-def _safe_evaluate(
```

Comment:
I'll need to partially revert this removal - `torch_max_mem` does not cover the fallback to cpu.